packages/hockmd/hockmd.0.1.0/opam

```diff
@@ -0,0 +1,45 @@
+opam-version: "2.0"
+synopsis: "A library to access hackmd's api"
+description: "A library to access hackmd's api"
+maintainer: ["Paul-Elliot Anglès d'Auriac <peada@free.fr>"]
+authors: ["Paul-Elliot Anglès d'Auriac <peada@free.fr>"]
+license: "ISC"
+homepage: "https://github.com/panglesd/hockmd"
+doc: "https://panglesd.github.io/hockmd-api/"
+bug-reports: "https://github.com/panglesd/hockmd/issues"
+depends: [
+  "ocaml" {>= "4.08.0"}
+  "dune"
+  "cohttp"
+  "cohttp-lwt-unix"
+  "lwt"
+  "ppx_deriving"
+  "yojson"
+  "ppx_yojson_conv"
+  "alcotest" {with-test}
+  "odoc" {with-doc}
+]
```

Comment:
To avoid issues with `Result` not being the same as `Stdlib.Result` the `result` library < 1.5 should be avoided
```suggestion
]
conflicts: [
  "result" {< "1.5"}
]
```